### PR TITLE
[RFC] misc: Fix some typos and trailing whitespace.

### PR DESCRIPTION
**Pull request by lucc**
These are some typos I found while doing other stuff.

### CODE REVIEW

File: test/functional/job/job_spec.lua [hunk 1]
```diff
@@ -119,7 +119,7 @@ describe('jobs', function()
     eq({'notification', 'exit', {0, 0}}, next_msg())
   end)
 
-  it('wont allow jobsend with a job that closed stdin', function()
+  it('won\'t allow jobsend with a job that closed stdin', function()
```
Comment (Pyrohh) [inline test/functional/job/job_spec.lua]: You could instead surround the string with double quotes; same goes for the other instances.

**Comment by Pyrohh** [comment]
Besides the nit, LGTM.

**Comment by Pyrohh** [comment]
LGTM

**Comment by justinmk** [comment]
Merged

**Pull request closed without merging**